### DISCUSSION TITLE
I am not able to assemble the layers in my web-site.

### DISCUSSION THREAD

**Discussion by jjpicturesahora** [community/community#61317] [Programming Help]
### Body

So I have this web site I am developing and already managed to set a beautiful picture as background and created a header (a div) for it which is a section bar of the same image placed at the top of the page where is possible to appreciate how they are both part of the same image but separate html elements. Now the problem I encounter is that when I place content inside the divs this content goes elsewhere and does not show where I want it. It's true that every html (the body and the header) have both an element inside but, Should I include one and only one element per container? It seems only natural to add in the same div as many elements as I see fit (for example, in the header place the clipped image I intend to use as background image and on top of it a logotype). Or should I place a transparent div on top of the other and every time add a single element per layer. Now, on top of that Who will be the parent of who? See my dilema, I am just trying to organize a simple web site with a few elements and I don't know how many elements (png imagers etc.) I am able to place per div and how to keep the content "inside" the div, not elsewhere. Please, somebody give me a hand. Thank you for your help!

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by DevRohit06** [comment]
Hello there @jjpicturesahora 
You can put as many element you want inside a section or a div, but when it comes to deciding the positions of the elements we should use css positioning, CSS Box Model, CSS Display and all. 
If you're struggling with a specific layout, you can provide more details about your code, and I'd be happy to provide more specific guidance.

**Reply by jjpicturesahora** [reply]
Hi there, sorry for the delay. Yes It would be most helpful to receive help! I feel kinda lost over here. In any case I haven't made any progress yet, but if I make any shall I still write to you? I would appreciate to take you up on your offer!

**Reply by DevRohit06** [reply]
for sure go ahead !!
